backend/src/workers/tasks.py

```
@@ -0,0 +1,177 @@
+"""
+import ujson
+import redis.asyncio as redis
+from src.workers.broker import broker, REDIS_URL
+
+
+@broker.task(task_name="run_agent_stream")
+async def run_agent_stream(
+    task_dict: dict,
+    user_id: str,
+    thread_id: str,
+    config_dict: dict,
```

Ruff (0.14.10)

Unused function argument: `config_dict`

(ARG001)